deploy/katello.spec

```diff
@@ -116,8 +116,9 @@ Summary:    Common runtime components of %{name}
 Requires:       wget
 Requires:       curl
 Requires:       %{?scl_prefix}rubygem-katello
-Requires:       rubygem-hammer_cli 
+Requires:       rubygem-hammer_cli
 Requires:       rubygem-hammer_cli_foreman
+Requires:       rubygem-hammer_cli_foreman_docker
```

Comment:
I don't think SAM gets docker so this may need to go in the katello RPM section. @thomasmckay